patches/third_party/devtools-frontend/src/front_end-panels-application-ApplicationPanelSidebar.ts.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
+index 9becc447d320e36196e7f475984d8b64899c0cdb..0838bd1ccc09d64190a52389fbfb7bdfadcccb4e 100644
+--- a/front_end/panels/application/ApplicationPanelSidebar.ts
++++ b/front_end/panels/application/ApplicationPanelSidebar.ts
+@@ -250,7 +250,7 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
+   interestGroupTreeElement: InterestGroupTreeElement;
+   databasesListTreeElement: ExpandableApplicationPanelTreeElement;
+   cookieListTreeElement: ExpandableApplicationPanelTreeElement;
+-  trustTokensTreeElement: TrustTokensTreeElement;
++  trustTokensTreeElement: TrustTokensTreeElement|undefined;
```

Comment:
oh, was this required because we commented out https://github.com/brave/brave-core/pull/17118/files#diff-23046df64bccd12252ff893d49474e6af4b2831de5d33efd50f3a8c4fc8c11c1R19 ? If so then go back to the original patch because we still end up with a modified line this way

Reply:
Sorry, the link seems to be broken(Github are bad at these 😭) . This line was required because we commented out the initialization of this property, which seems to be what you linked. Reverted back the very original patch - commenting out `appendChild()`